src/parameterizations/lateral/MOM_Zanna_Bolton.F90

```diff
@@ -432,7 +466,12 @@ subroutine ZB2020_lateral_stress(u, v, h, diffu, diffv, G, GV, CS, &
 
   ! Compute the stress tensor given the
   ! (optionally sharpened) velocity gradients
-  call compute_stress(G, GV, CS)
+  if (CS%use_ann==0) then
```

Comment:
I see here that `USE_ANN` is not a boolean, so perhaps `ZB2020_ANN_MODE` would be a better parameter name?